fighters/lucina/src/acmd/specials.rs

```diff
@@ -1330,10 +1336,11 @@ unsafe fn lucina_special_air_s4_hi_effect(fighter: &mut L2CAgentBase) {
 unsafe fn lucina_special_s4_s_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
+    let add_damage = 0.5*(VarModule::get_float(boma.object(), vars::lucina::instance::NEW_DAMAGE) - VarModule::get_float(boma.object(), vars::lucina::instance::CURRENT_DAMAGE));
```

Comment:
For the riposte swings, I feel like the global const `NEW_DAMAGE` is a bit superfluous, you could probably just call `DamageModule::damage(boma, 0)` in the local variable definition. Other than that, your implementation looks like it works.
```suggestion
    let add_damage = 0.5*(DamageModule::damage(boma, 0) - VarModule::get_float(boma.object(), vars::lucina::instance::CURRENT_DAMAGE));
```